buildSrc/src/main/kotlin/common-conventions.gradle.kts

```
@@ -68,20 +70,27 @@ node {
 }
 
 spotless {
+    val npmExec = when(System.getProperty("os.name".toLowerCase()).contains("windows")) {
```

The string is already lowercase. I think you mean to lowercase the result of getProperty():
```suggestion
    val npmExec = when(System.getProperty("os.name").toLowerCase().contains("windows")) {
```

---

oof good catch